src/KKS_BrowserFolders_Hooks/Overlord.cs

```diff
@@ -20,6 +20,61 @@ public static void Init()
             Harmony.CreateAndPatchAll(typeof(Overlord));
         }
 
+        //In case of duplicate names in different folders
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(File), nameof(File.Move), typeof(string), typeof(string))]
+        internal static void MoveHook(string sourceFileName, string destFileName)
+        {
+            if(File.Exists(destFileName))
+                File.Delete(destFileName);
+        }
```

Comment:
This hook is kind of an overkill, since it applies to the whole game and can cause unexpected behavior in other plugins or game code. It would be better to handle the paths correctly so that there are subfolders in the old folder. I'll fix it up after merging.